src/libraries/System.Data.Common/src/System/Data/SqlDbType.cs

```diff
@@ -3,40 +3,185 @@
+        /// <see cref="byte"/>. An 8-bit unsigned integer.
+        /// </summary>
         TinyInt = 20,
+
+        /// <summary>
+        /// <see cref="System.Array"/> of type <see cref="byte"/>. A variable-length stream of binary data ranging between 1 and 8,000 bytes.
+        /// Implicit conversion fails if the byte array is greater than 8,000 bytes. Explicitly set the object when working with byte arrays
+        /// larger than 8,000 bytes.
+        /// </summary>
         VarBinary = 21,
+
+        /// <summary>
+        /// <see cref="string"/>. A variable-length stream of non-Unicode characters ranging between 1 and 8,000 characters.
+        /// Use <see cref="F:System.Data.SqlDbType.VarChar"/> when the database column is <see langword="varchar(max)"/>.
+        /// </summary>
         VarChar = 22,
+
+        /// <summary>
+        /// <see cref="object"/>. A special data type that can contain numeric, string, binary, or date data as well as the SQL Server values Empty and Null,
+        /// which is assumed if no other type is declared.
+        /// </summary>
         Variant = 23,
+
+        /// <summary>
+        /// An XML value. Obtain the XML as a string using the <see cref="M:Microsoft.Data.SqlClient.SqlDataReader.GetValue(System.Int32)"/> method or
+        /// <see cref="P:System.Data.SqlTypes.SqlXml.Value"/> property, or as an <see cref="T:System.Xml.XmlReader"/> by calling the
+        /// <see cref="M:System.Data.SqlTypes.SqlXml.CreateReader"/> method.
+        /// </summary>
         Xml = 25,
+
+        /// <summary>
+        /// A SQL Server user-defined type (UDT).
+        /// </summary>
         Udt = 29,
+
+        /// <summary>
+        /// A special data type for specifying structured data contained in table-valued parameters.
+        /// </summary>
         Structured = 30,
+
+        /// <summary>
+        /// Date data ranging in value from January 1,1 AD through December 31, 9999 AD.
+        /// </summary>
         Date = 31,
+
+        /// <summary>
+        /// Time data based on a 24-hour clock. Time value range is 00:00:00 through 23:59:59.9999999 with an accuracy of 100 nanoseconds.
+        /// Corresponds to a SQL Server <see langword="time"/> value.
+        /// </summary>
         Time = 32,
+
+        /// <summary>
+        /// Date and time data. Date value range is from January 1,1 AD through December 31, 9999 AD.
+        /// Time value range is 00:00:00 through 23:59:59.9999999 with an accuracy of 100 nanoseconds.
+        /// </summary>
         DateTime2 = 33,
+
+        /// <summary>
+        /// Date and time data with time zone awareness. Date value range is from January 1,1 AD through December 31, 9999 AD.
+        /// Time value range is 00:00:00 through 23:59:59.9999999 with an accuracy of 100 nanoseconds. Time zone value range
+        /// is -14:00 through +14:00.
+        /// </summary>
         DateTimeOffset = 34,
+
+        /// <summary>
+        /// A Json Value.
```

Comment:
For consistency with the other comments, for example, XML.

```suggestion
        /// A JSON value.
```